### PR TITLE
Readme update

### PR DESCRIPTION
**Pull request by fr1kin**
Update the README to be more db friendly

### CODE REVIEW

**Comment by popbob** [comment]
🤡

File: README.md [hunk 1]
```diff
@@ -31,6 +31,8 @@ I recommend allocating more ram in both environments as the Markers mod requires
 
 ##### config.properties tokens are not applied
 
+Sometimes when building, the config.properties resource will not be tokenized. This will break the tweaker because it wont know what mapping version to use. This can be fixed by cleaning your gradle and IDE build output and running setupDecompWorkspace again.
```
Comment (popbob) [inline README.md]: 👎🏿

File: README.md [hunk 2]
```diff
@@ -31,6 +31,8 @@ I recommend allocating more ram in both environments as the Markers mod requires
...
+
```
Comment (popbob) [inline README.md]: 👎🏿

File: README.md [hunk 3]
```diff
@@ -1,6 +1,6 @@
 # ForgeHax
 [![Build Status](https://jenkins.nhackindustries.com/buildStatus/icon?job=ForgeHax/master)](https://jenkins.nhackindustries.com/job/ForgeHax/job/master)
-[![](https://img.shields.io/matrix/forgehax%3Anerdsin.space.svg?logo=matrix)](https://matrix.to/#/!RBkfmpSEFHJliToAea:nerdsin.space?via=nerdsin.space&via=matrix.org)
+[![](https://img.shields.io/matrix/forgehax:nerdsin.space.svg?label=%23forgehax%3Anerdsin.space&logo=matrix)](https://matrix.to/#/#forgehax:nerdsin.space)
```
Comment (popbob) [inline README.md]: 👎🏿

File: README.md [hunk 4]
```diff
@@ -1,9 +1,7 @@
 # ForgeHax
+[![](https://img.shields.io/badge/download-latest-blue.svg?logo=java)](https://jenkins.nhackindustries.com/job/ForgeHax/job/master/lastSuccessfulBuild/)
```
Comment (popbob) [inline README.md]: 👎🏿

File: README.md [hunk 5]
```diff
@@ -3,54 +3,74 @@
 [![Build Status](https://jenkins.nhackindustries.com/buildStatus/icon?job=ForgeHax/master)](https://jenkins.nhackindustries.com/job/ForgeHax/job/master)
 [![](https://img.shields.io/matrix/forgehax:nerdsin.space.svg?label=%23forgehax%3Anerdsin.space&logo=matrix)](https://matrix.to/#/#forgehax:nerdsin.space)
 
-A Minecraft cheat that runs as a Forge mod
+A Minecraft cheat that runs as a Forge mod.
```
Comment (popbob) [inline README.md]: 👎🏿

File: README.md [hunk 6]
```diff
@@ -3,54 +3,74 @@
...
 
-Since ForgeGradle has updated to 2.3, Java 8 Version 131 (lastest as of writing this) is required. Make sure your JAVA_HOME is set to this version of the JDK. If you don't, setupDecompWorkspace will fail at recompileMc.
+## Installing
```
Comment (popbob) [inline README.md]: 👎🏿

File: README.md (hunk 2 above)
Comment (fr1kin) [reply README.md]: lose some weight

File: README.md [hunk 7]
```diff
@@ -3,54 +3,74 @@
...
+
+1. Download the latest version of [Minecraft Forge](https://files.minecraftforge.net/) for the corresponding ForgeHax Minecraft version (this is important if you want to run older versions of ForgeHax).
+2. Download the [latest ForgeHax build](https://jenkins.nhackindustries.com/job/ForgeHax/job/master/lastSuccessfulBuild/). Do NOT install the jar that ends with `-sources.jar`. That one contains the source code and isn't compiled.
+3. Place the ForgeHax jar into the `.minecraft/mods/` directory. If you want to organize by Minecraft version, you can place it under `.minecraft/mods/{mc.version}` where `mc.version` is the version of Minecraft running (ex: `.minecraft/mods/1.12.2`).
+4. Launch Minecraft using the Forge profile. ForgeHax should now be loaded.
+
+## Wiki
```
Comment (popbob) [inline README.md]: 👎🏿

File: README.md [hunk 8]
```diff
@@ -3,54 +3,74 @@
...
+
+If you need any help, please check the [ForgeHax Wiki](https://github.com/fr1kin/ForgeHax/wiki) before submitting an issue.
+
+## Known Issues
```
Comment (popbob) [inline README.md]: 👎🏿

File: README.md [hunk 9]
```diff
@@ -3,54 +3,74 @@
...
+
+World Downloader Forge is a mod known to wreak havoc on ForgeHax's asm tweaker. There is some race condition it has with Mixin causing ForgeHax to only successfully transform classes sometimes. Luckly this is the only known mod known to cause this issue, and other Mixin based tweakers work fine with ForgeHax (i.e Liteloader, Future). 
+
+## Building
```
Comment (popbob) [inline README.md]: 👎🏿

File: README.md [hunk 10]
```diff
@@ -3,54 +3,74 @@
...
+Java 8 Version 131+ is required. Make sure your JAVA_HOME is set to this version of the JDK. If you don't, setupDecompWorkspace will fail at recompileMc.
 
 You should allocate more ram when running `setupDecompWorkspace` (around 4GB should be ok)
 
-I recommend allocating more ram in both environments as the BlockEsp mod requires a lot of space for the vertex buffers. I might get around to fixing this if it becomes too big of an issue.
+I recommend allocating more ram in both environments as the Markers mod requires a lot of memory for the vertex buffers. I might get around to fixing this if it becomes too big of an issue.
```
Comment (popbob) [inline README.md]: 👎🏿

File: README.md [hunk 11]
```diff
@@ -3,54 +3,74 @@
...
+
+#### Common build issues
```
Comment (popbob) [inline README.md]: 👎🏿

File: README.md [hunk 12]
```diff
@@ -3,54 +3,74 @@
...
+
+##### config.properties tokens are not applied
 
-# FAQ
-_How do I install ForgeHax?_
+## FAQ
+#### How do I install ForgeHax?
```
Comment (popbob) [inline README.md]: 👎🏿

File: README.md [hunk 13]
```diff
@@ -3,54 +3,74 @@
...
 
 Download [Minecraft Forge](https://files.minecraftforge.net/) and put the ForgeHax jar into .minecraft/mods
 
-_How do I use commands?_
+#### How do I use commands?
```
Comment (popbob) [inline README.md]: 👎🏿

File: README.md [hunk 14]
```diff
@@ -3,54 +3,74 @@
...
 
-_How do I enter an argument that contains spaces?_
+#### How do I enter an argument that contains spaces?
 
 Use quotes.
 
 Example: `.chatbot spam add spam "This will be treated as one argument"`
 
-_How do I see a list of mods?_
+#### How do I see a list of mods?
```
Comment (popbob) [inline README.md]: 👎🏿

**Pull request merged**